framework/PageInputs/PageMultiSelect.cy.tsx

```diff
@@ -5,11 +5,13 @@ import { PageMultiSelect } from './PageMultiSelect';
 import { PageSelectOption } from './PageSelectOption';
 
 interface ITestObject {
+  id: number;
```

Comment:
It would be interesting to provide an isOptionValueEqual prop to explore that one.